config/crd/bases/metal3.io_hardwareclassifications.yaml

```diff
@@ -40,108 +46,90 @@ spec:
         spec:
           description: HardwareClassificationSpec defines the desired state of HardwareClassification
           properties:
-            hardwareCharacteristics:
-              description: HardwareCharacteristics defines expected hardware configurations
-                for Cpu, Disk, Nic and Ram.
+            expectedValidationConfiguration:
```

Comment:
Did you mean to rename this field?

Reply:
@Ashughorla  Please refer PR #21 , We have finalized hardwareCharacteristics. So don't rename it.

Reply:
@digambar15 yes sure, i will revert it back.